otter/convergence/service.py

```diff
@@ -180,13 +180,15 @@ def all_met(server, current_lb_nodes):
                              if node.matches(server)]))
 
 
-def update_servers_cache(group, now, servers, lb_nodes, include_deleted=True):
+def update_servers_cache(group, now, servers, lb_nodes, lbs,
+                         include_deleted=True):
     """
     Updates the cache, adding servers, with a flag if autoscale is active on
     each one.
     :param group: scaling group
     :param list servers: list of NovaServer objects
     :param list lb_nodes: list of CLBNode objects
+    :param dict lbs: load balancer objects keyed on ID (currently ignored)
```

Comment:
I was also a little bit confused by "currently ignored" since this is supposed to be the _last_ bit of #1870.  So let me explain what I discovered to confirm my understanding:

`convergence_exec_data` gathers "resources", which are passed as keyword arguments to `executor.update_cache`.  For servers, `update_cache` is `get_all_launch_server_data`, which now returns a `lbs` key; this means that `convergence_exec_data` must accept this argument.  But `get_all_launch_server_data` is returning this for the benefit of `launch_server_executor.plan`, by which I mean of course `plan_launch_server`, which now requires `lbs` to do _its_ job.

For this change, I think the only thing to do is to note where `lbs` is actually coming from and why it's unused (since as far as I can tell the plan is not to use it in the future) in the docstring here.  In the future, I hope we can do a little refactoring to make it more structurally obvious how data moves around here.